README.md

```diff
@@ -0,0 +1,86 @@
+# auxtools
+
+auxtools is a Rust-based successor to the [Extools](https://github.com/MCHSL/extools) project.
+
+Currently, it implements some of the features such as the debug server, but is planned to supplant most functionality (maptick, etc.).
+
+## Features
+Debug Server - Working with SpaceManiac's [SpacemanDMM](https://github.com/SpaceManiac/SpacemanDMM), Auxtools interfaces with the debugger frontend to provide debugger information and management of breakpoints.
+
+*Dissassembly*  - When currently stopped at a breakpoint, enter the `#dis` command into the Debug Console in VSC to see the DM bytecode for the current proc. You can also provide it any arbitrary proc path like `#dis /mob/proc/Life`.
+
+## Dependencies
+
+*These instructions were taken directly from tgstation's [rust-g](https://github.com/tgstation/rust-g) documentation.*
+
+The [Rust] compiler:
+
+1. Install the Rust compiler's dependencies (primarily the system linker):
+
+   * Ubuntu: `sudo apt-get install gcc-multilib`
+   * Windows (MSVC): [Build Tools for Visual Studio 2017][msvc]
+   * Windows (GNU): No action required
+
+1. Use [the Rust installer], or another Rust installation method,
+   or run the following:
+
+    ```sh
+    curl https://sh.rustup.rs -sSfo rustup-init.sh
+    chmod +x rustup-init.sh
+    ./rustup-init.sh
+    ```
+
+1. Set the default compiler to **32-bit**:
+
+    ```sh
+    # Clone the `auxtools` repository to a directory of your choice
+    git clone https://github.com/willox/auxtools
+    # in the `auxtools` directory...
+    cd auxtools
+    # Linux
+    rustup target add i686-unknown-linux-gnu
+    # Windows
+    rustup target add i686-pc-windows-msvc
+    ```
+
+System libraries:
+
+* Ubuntu and Debian users run:
+
+    ```sh
```

Comment:
I don't know that these dependencies are required for auxtools. Our CI is just doing:
```sh
sudo dpkg --add-architecture i386
sudo apt-get update
sudo apt install build-essential g++-multilib libc6-i386 libstdc++6:i386
```